hosts/juno/junocam.py

```diff
@@ -52,51 +50,43 @@ def from_file(filespec, fast_distortion=True,
     Juno.load_spks(meta.tstart0, meta.tstart0 + 3600.)
 
     # Construct a Pushframe for each framelet
-
     snap = False
-
-
     obs = []
     for i in range(meta.nframelets):
         fmeta = Metadata(flabels[i])
 
         if snap:
-            item = (oops.obs.Snapshot(("v","u"),
-                                 (fmeta.tstart, fmeta.tdi_texp), fmeta.fov,
-                                 "JUNO", "JUNO_JUNOCAM",
-                                 instrument = "JUNOCAM",
-                                 filter = fmeta.filter,
-                                 data = framelets[:,:,i]))
-
-
+            item = (oops.obs.Snapshot(
+                                ('v','u'),
+                                (fmeta.tstart, fmeta.tdi_texp), fmeta.fov,
+                                'JUNO', 'JUNO_JUNOCAM',
+                                instrument = 'JUNOCAM',
+                                filter = fmeta.filter,
+                                data = framelets[:,:,i]))
```

Comment:
Remove the surrounding parentheses.

Reply:
The extraneous parentheses are still there. Also, BTW, when you have this many arguments in a call, I recommend you use "keyword=value" for more of them. For example, I have no idea what 'JUNO' and 'JUNO_JUNOCAM' mean in this list. As I have learned the hard way, a call like this is likely to break if anything about the call signature ever changes.